sdk/python/feast/infra/materialization/contrib/bytewax/bytewax_materialization_engine.py

```diff
@@ -197,9 +197,12 @@ def _materialize_one(
                     logger.info(f"{feature_view.name} materialization still running...")
                     sleep(30)
                 logger.info(f"{feature_view.name} materialization complete with status {job.status()}")
-            except (KeyboardInterrupt, BaseException) as e:
+            except BaseException as e:
                 logger.info(f"Killing job {job.job_id()}")
-                self.batch_v1.delete_namespaced_job(job.job_id(), self.namespace)
+                try:
+                    self.batch_v1.delete_namespaced_job(job.job_id(), self.namespace)
+                except ApiException as de:
+                    logger.warning(f"Could not delete job due to API Error: {ae.body}")
```

Comment:
I think here we could also do re-raise, i.e.
```
except {}:
    logger.warning...
    raise
```

Reply:
Could, but we're already about to re-raise the first exception, which I think is more valuable to see.